.github/ISSUE_TEMPLATE/4-release.yml

```diff
@@ -0,0 +1,124 @@
+name: 'WooCommerce: release checklist'
+description: Create a new issue to track the next WooCommerce core release progress.
+title: "WooCommerce RELEASE_VERSION: RC, final and dot-releases progress tracking"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This issue provides visibility on the progress of the release process of WooCommerce core RELEASE_VERSION and 
+        centralizes any conversations about it. The ultimate goal of this issue is to keep a reference of the steps, 
+        resources, work, and conversations about this release so it can be helpful for the next contributors releasing 
+        a new WooCommerce core version.
+
+        -   WooCommerce core version to release: RELEASE_VERSION ([milestone](https://github.com/woocommerce/woocommerce/milestone/MILESTONE_ID))
+        -   Release Lead: RELEASE_LEAD
+        -   Release Date: RC release on RELEASE_DATE
+        -   Release Date: final release on RELEASE_DATE
+        -   WooCommerce core change log: [here](https://github.com/woocommerce/woocommerce/blob/trunk/changelog.txt)
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Resources
+
+        -   [Release calendar](https://developer.woocommerce.com/release-calendar/)
+        -   [Previous releases](https://developer.woocommerce.com/releases/)
+        -   Latest (version 9.9+) [release process changes](https://developer.woocommerce.com/2025/03/05/upcoming-changes-to-the-woocommerce-release-process/)
+        -   Internal detailed reference to the release process: Pf59Ax-7F-p2
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Checklist
+
+  - type: markdown
+    attributes:
+      value: |
+        ### RC release
+        
+        #### Get release artifact ready
+
+        -    [] Ensure all [CFEs](https://github.com/woocommerce/woocommerce/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22code%20freeze%20exception%22) for the target release version are addressed and merged
+        -    [] Validate stable tag in `readme.txt` for the release branch and update if necessary
+        -    [] Validate stable tag in `readme.txt` for the SVN `trunk` ([here](https://plugins.svn.wordpress.org/woocommerce/trunk/readme.txt))
+        -    [] Skip changelog generation (?)
```

Comment:
Don't we need this step for RCs? 🤔

Reply:
I'm not sure. @jorgeatorres - can you help clarify those steps in the context of RCs?

Reply:
Yes, we need to generate the changelog for all releases. The overall release process is identical for RCs and also the final release, except for the monitoring part.

Reply:
Gotcha, I'll update this entry then.

Reply:
Updated in f2294914cc82d4e5ce925efd77e465a3ab5596b1